modules/local/rename_raw_data_files.nf

```diff
@@ -5,7 +5,7 @@ process RENAME_RAW_DATA_FILES {
     conda "conda-forge::sed=4.7"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
-        'docker.io/ubuntu:20.04' }"
+        'nf-core/ubuntu:20.04' }"
```

Comment:
The docker.io was specifically mentioned in https://nfcore.slack.com/archives/C04QR0T3G3H/p1682587842705369, why the change to nf-core?

Reply:
Actually not really a need there, but we have this one on quay.io, and they're less picky than docker.io

Reply:
Thanks for the info!